backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationMutationResolver.java

```diff
@@ -62,13 +62,13 @@ public ApiFacility addFacility(
         _avs.getValidatedAddress(
             street, streetTwo, city, state, zipCode, _avs.FACILITY_DISPLAY_NAME);
     StreetAddress providerAddress =
-        _avs.getValidatedAddress(
-            orderingProviderStreet,
-            orderingProviderStreetTwo,
-            orderingProviderCity,
-            orderingProviderState,
-            orderingProviderZipCode,
-            _avs.PROVIDER_DISPLAY_NAME);
+        new StreetAddress(
```

Comment:
This code seems like it's probably repeated someplace, so we should factor it out, sometime that is not now.